src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -492,12 +534,14 @@ uct_posix_mem_alloc(uct_md_h tl_md, size_t *length_p, void **address_p,
             goto err_close;
         }
 
+        has_ns = !ucs_sys_ns_is_default(UCS_SYS_NS_TYPE_PID) ||
+                 !ucs_sys_ns_is_default(UCS_SYS_NS_TYPE_USER);
```

Comment:
do we really have to pass user NS in the address?
If PID ns is the same, couldn't we try to open the remote file and if we could not open (for any reason) consider the remote iface as not reachable? or use access(2) to check if have permissions to open?

Reply:
the remote file contains the pid and peer_fd, and it is available doing unpack so it seems it arrives after the is reachable check/lane is setup.

Reply:
as discussed, now checking directly the capability to posix open